app.rb

```diff
@@ -116,7 +114,6 @@ def create_a_book
     book = Book.new(title, author)
     @book.push(book)
     puts 'Book cretaed successfully'
-    puts
     choose_a_number
   end
```

Comment:
- [ ] in the `create_a_book` method you are printing to the console getting information from the user and saving the information this is not in line with the single responsibility rule, have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more information.